src/pysamp/pygamemode.cpp

```diff
@@ -31,6 +31,12 @@ PyGamemode::PyGamemode(const char * path)
 		sampgdk::logprintf("Setting python workspace failed.");
 
 	PyList_Append(sysPath, PyUnicode_FromString(absolute));
+
+	PyObject *module = PyImport_ImportModule("pysamp");

```

Comment:
What is the intent behind this? Pysamp is the module we provide through the plugin so it should always correctly load.

Reply:
Yep - if we're not loading it, the sys.stdout override wouldn't happen, because otherwise `PyInit_samp` isn't called (as I explained on Discord) :smile: